src/coffee-script.coffee

```diff
@@ -19,6 +19,11 @@ exports.FILE_EXTENSIONS = ['.coffee', '.litcoffee', '.coffee.md']
 # Expose helpers for testing.
 exports.helpers = helpers
 
+# Function that allows for btoa in both nodejs and the browser.
+toASCII = (src) ->
+  if typeof window isnt 'undefined' then btoa(src)
```

Comment:
This is the wrong kind of test. You should test for the existence of `btoa`, not assume it is there. Same with `Buffer`. If neither are there, give a nice error.